doc/releases/changelog-dev.md

```diff
@@ -45,7 +45,25 @@
   ```
 
 * New basis sets, `6-311g` and `CC-PVDZ`, are added to the qchem basis set repo.
-  [#3279](https://github.com/PennyLaneAI/pennylane/pull/3279)
+  [(#3279)](https://github.com/PennyLaneAI/pennylane/pull/3279)
```

Comment:
We should stick with one and change it everywhere in the file

Reply:
I think this is the more standard notation,

Reply:
Could you change it everywhere in the file? This way we will avoid confusions.

Reply:
Added 👍🏼